enderio-base/src/main/java/com/enderio/base/client/renderers/GearBEWLR.java

```diff
@@ -0,0 +1,41 @@
+package com.enderio.base.client.renderers;
+
+import com.enderio.base.EnderIO;
+import com.enderio.core.common.util.PoseStackHelper;
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.mojang.math.Vector3f;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.model.geom.EntityModelSet;
+import net.minecraft.client.renderer.BlockEntityWithoutLevelRenderer;
+import net.minecraft.client.renderer.MultiBufferSource;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.client.renderer.block.model.ItemTransforms.TransformType;
+import net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher;
+import net.minecraft.client.resources.model.BakedModel;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+
+public class GearBEWLR extends BlockEntityWithoutLevelRenderer{
+	
+	public static final GearBEWLR INSTANCE = new GearBEWLR(Minecraft.getInstance().getBlockEntityRenderDispatcher(), Minecraft.getInstance().getEntityModels());
+	private float tpr = 120; //ticks per rotation
```

Comment:
Please add a todo here for making rotation speed configurable :) I'll look into that in the future if you aren't going to in this PR